G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google.Tests/Commands/OcrGoogle.Find/ShouldReturnRectanglePosition_WhenTextIsFound.G1ANT

```diff
@@ -1,13 +1,15 @@
 ﻿addon iexplorer version 2.205.17304.1000
 addon ocrgoogle version 2.205.17304.1000
 
+process.kill iexplore
 ie.open http://digitalnativestudios.com/textmeshpro/docs/rich-text/lowercase-uppercase-smallcaps.png
 window ‴✱Internet Explorer✱‴ style maximize
+delay milliseconds 200
```

Comment:
Why 200, and not 2 or 2000? No value will be good enough if you won't wait for the content to be loaded.

Reply:
Its enough to let window maximize, can't OCR text on window that is currently resizing.
Test is about OCR not about window resizing.